src/language-js/parse/babel.js

```diff
@@ -36,7 +36,7 @@ const parseOptions = {
     "throwExpressions",
     "partialApplication",
     "decorators",
-    "importAssertions",
+    "importAttributes",
```

Comment:
I don't think we can drop support for assertions, Node.js already support import json file with that syntax, so people already use that syntax. Correct me if I'm wrong.

Reply:
You are right. However, Import Assertions was Stage 3 and was dropped from TC39. Therefore, it would make sense for us to drop its syntax. And I think v3 is a good opportunity to do so.

Reply:
Import assertions will stick around for a while, probably at least until the last Node.js version without support for import attributes will reach EOL.

I wish tools encouraged users to migrate to attributes as soon as possible, but it might be too early for this breaking change.

Reply:
Agree, we should be able to format runnable code.

Reply:
Thank you @fisker and @nicolo-ribaudo. I got it.